src/pug/includes/body.pug

```diff
@@ -43,7 +43,7 @@ include ./fork.pug
 				- difficulty = swag.difficulty + ' difficulty';
 				- difficultyTitle = 'Difficulty: ' + swag.difficulty;
 				div(
-					class="item visible " + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
+					class="item " + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
```

Comment:
This also works:

```suggestion
					class="item " + (!swag.expired ? ' visible' : '') + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
```

But it seems like it doesn't make the promos load any faster (for me, at least).

Reply:
I don't think we should worry too much about TTI in this specific case. We have a few other things to improve (most notably how we handle third party scripts) before we should handle stuff like this.

Disabling asset optimization on Netlify's side dropped TTI from 3.7 to 2.7 seconds for me so we're definitely not far from 🔥